packages/server/src/models/UserModel.ts

```diff
@@ -28,6 +28,8 @@ import changeEmailNotificationTemplate from '../views/emails/changeEmailNotifica
 import { NotificationKey } from './NotificationModel';
 import prettyBytes = require('pretty-bytes');
 import { Env } from '../utils/types';
+import config from '../config';
```

Comment:
We avoid importing the complete config in the models now, to keep things more modular. Instead, in the constructor, you can copy the dependencies you will need. See the constructor of ItemModel as an example

Reply:
implemented as suggested